frontend/integration-tests/tests/service-catalog/service-binding.scenario.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-undef, no-unused-vars */
+
+import {browser, $, $$, ExpectedConditions as until, by} from 'protractor';
+
+import { appHost, checkLogs, checkErrors, testName } from '../../protractor.conf';
+import * as srvCatalogView from '../../views/service-catalog.view';
+import * as sidenavView from '../../views/sidenav.view';
+import * as crudView from '../../views/crud.view';
+import {execSync} from "child_process";
+
+describe('Test for Cluster Service Binding', () => {
+  beforeAll(async() => {
+    browser.get(`${appHost}/status/ns/${testName}`);
+    await browser.wait(until.presenceOf($('#sidebar')));
+  });
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('creates a new binding for new service instance `mariadb-persistent`', async() => {
+    await sidenavView.clickNavLink(['Service Catalog', 'Service Classes']);
+    await srvCatalogView.isLoaded();
+
+    const mariaDBLink = srvCatalogView.serviceClassLinkFor('MariaDB');
+    expect(mariaDBLink.isDisplayed()).toBe(true);
+
+    await mariaDBLink.click();
+    await srvCatalogView.isLoaded();
+
+    expect(srvCatalogView.createInstanceButton.isDisplayed()).toBe(true);
+    srvCatalogView.createInstanceButton.click();
+    await srvCatalogView.createInstanceFormIsLoaded();
+
+    // select 'default' namespace, then submit form
```

Comment:
The tests really shouldn't be putting things in the default namespace, which is a system namespace on the cluster and can potentially break things. We should create and use a test namespace.